src/Model/PhasorDynamics/Governor/Tgov1/Tgov1Impl.hpp

```diff
@@ -236,11 +243,15 @@ namespace GridKit
       }
 
       /**
-       * @brief Residuals of system equations evaluated locally
+       * @brief Internal residuals
        *
        */
       template <class ScalarT, typename IdxT>
-      __attribute__((always_inline)) int Tgov1<ScalarT, IdxT>::evaluateResidualLocally(ScalarT* y, ScalarT* yp, ScalarT* f)
+      __attribute__((always_inline)) int Tgov1<ScalarT, IdxT>::evaluateInternalResidual(
```

Comment:
```suggestion
      __attribute__((always_inline)) inline int Tgov1<ScalarT, IdxT>::evaluateInternalResidual(
```
Could consider a macro for this